applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -16,9 +16,6 @@ config BT_ECC
 config BT_EXT_ADV
 	default y
 
-config BT_DEVICE_NAME
-	default "NRF5340_AUDIO"
-
 # Mandatory to support at least 1 for ASCS
 config BT_ATT_PREPARE_COUNT
 	default 1
```

Comment:
Code below related to unicast should be moved to bt_stream/unicast/Kconfig.default